patroni/ctl.py

```diff
@@ -515,7 +524,13 @@ def reload(obj, cluster_name, member_names, force, role):
 def restart(obj, cluster_name, member_names, force, role, p_any, scheduled, version, pending, timeout):
     cluster = get_dcs(obj, cluster_name).get_cluster()
 
-    members = get_members(cluster, cluster_name, member_names, role, force, 'restart')
+    if scheduled is None and not force:
+        now = (datetime.datetime.now() + timedelta(hours=1)).replace(second=0, microsecond=0).isoformat()
```

Comment:
```python
(datetime.datetime.now() + datetime.timedelta(hours=1)).strftime('%Y-%m-%dT%H:%M')
```

and you can drop import from the [line 39](https://github.com/zalando/patroni/pull/1119/files#diff-b558c390fbbb65333f66b7b6a8fe73cdR39)